pkg/util/azureclient/fake/deployments.go

```diff
@@ -3,15 +3,14 @@ package fake
 import (
```

Comment:
would like to see a more or less 1-1 map between the files and structs in azureclient/ and those in azureclient/fake